eval/src/vespa/eval/eval/tensor_function.cpp

```diff
@@ -143,6 +166,67 @@ Map::compile_self(Stash &) const
     return Instruction(op_tensor_map, to_param(_function));
 }
 
+vespalib::string
+name_of(map_fun_t fun)
+{
+    if (fun == operation::Neg::f) return "-";
+    if (fun == operation::Not::f) return "!";
+    if (fun == operation::Cos::f) return "cos";
+    if (fun == operation::Sin::f) return "sin";
+    if (fun == operation::Tan::f) return "tan";
+    if (fun == operation::Cosh::f) return "cosh";
+    if (fun == operation::Sinh::f) return "sinh";
+    if (fun == operation::Tanh::f) return "tanh";
+    if (fun == operation::Acos::f) return "acos";
+    if (fun == operation::Asin::f) return "asin";
+    if (fun == operation::Atan::f) return "atan";
+    if (fun == operation::Exp::f) return "exp";
+    if (fun == operation::Log10::f) return "log10";
+    if (fun == operation::Log::f) return "log";
+    if (fun == operation::Sqrt::f) return "sqrt";
+    if (fun == operation::Ceil::f) return "ceil";
+    if (fun == operation::Fabs::f) return "fabs";
+    if (fun == operation::Floor::f) return "floor";
+    if (fun == operation::IsNan::f) return "isnan";
+    if (fun == operation::Relu::f) return "relu";
+    if (fun == operation::Sigmoid::f) return "sigmoid";
+    if (fun == operation::Elu::f) return "elu";
+    return "[other map function]";
+}
+
+vespalib::string
+name_of(join_fun_t fun)
```

Comment:
Consider have a common mapping from function to name to avoid duplication in dump_target.cpp.